local-interchain/interchain/genesis.go

```diff
@@ -62,7 +62,7 @@ func SetupGenesisWallets(config *types.Config, chains []ibc.Chain) map[ibc.Chain
 			for _, coin := range amount {
 				additionalWallets[chainObj] = append(additionalWallets[chainObj], ibc.WalletAmount{
 					Address: acc.Address,
-					Amount:  coin.Amount.Int64(),
+					Amount:  coin.Amount,
```

Comment:
Are we sure this compiles? IIRC, I had to add the int64 method to fix a compile error. Ideally, CI catches any compile problems but it was missed last time.

Reply:
The new GH action should catch any compile errors and fail.

Reply:
FYI, this is the PR that made the change Nix is referring to. 
https://github.com/strangelove-ventures/interchaintest/pull/679

Off the top of my head, I'm not sure how this affects local-ic.

Reply:
It was just a compile error, no logic broke. This PR fixes that and also ensures it does not happen again (build CI in this branch)